receiver/pluginreceiver/go.mod

```diff
@@ -42,18 +42,18 @@ require (
 	github.com/power-devops/perfstat v0.0.0-20210106213030-5aafc221ea8c // indirect
 	github.com/prometheus/client_golang v1.12.2 // indirect
 	github.com/prometheus/client_model v0.2.0 // indirect
-	github.com/prometheus/common v0.34.0 // indirect
+	github.com/prometheus/common v0.35.0 // indirect
 	github.com/prometheus/procfs v0.7.3 // indirect
 	github.com/prometheus/statsd_exporter v0.21.0 // indirect
 	github.com/shirou/gopsutil/v3 v3.22.6 // indirect
-	github.com/spf13/cobra v1.4.0 // indirect
+	github.com/spf13/cobra v1.5.0 // indirect
```

Comment:
Where do you find the documentation for updating this dependency and dependencies like it that don't seem to be listed in the changelog of the v0.55.0 release?

Reply:
So does it get changed upon `go mod tidy`?

Reply:
Yeah, after updating the dependency `go mod tidy` takes care of this.